packages/zowe-explorer/webpack.config.js

```diff
@@ -84,11 +74,19 @@ const config = {
                             compilerOptions: {
                                 sourceMap: true,
                             },
-                            projectReferences: true,
```

Comment:
Feel free to correct me if I'm missing something, but I think we'll still want `projectReferences: true` to keep @t1m0thyj's enhancement to the `watch` script from [this commit](https://github.com/zowe/zowe-explorer-vscode/commit/8addc7ec4a35337625e59b1ea95ecafe96509ce2).